composer.json

```diff
@@ -72,97 +71,98 @@
-        "symfony/mailer": "^5.4 || ^6.0 || ^7.0",
-        "symfony/mime": "^5.4 || ^6.0 || ^7.0",
-        "symfony/options-resolver": "^5.4 || ^6.0 || ^7.0",
-        "symfony/process": "^5.4 || ^6.0 || ^7.0",
-        "symfony/property-access": "^5.4 || ^6.0 || ^7.0",
+        "symfony/asset": "^6.4 || ^7.1",
+        "symfony/cache": "^6.4 || ^7.1",
+        "symfony/config": "^6.4 || ^7.1",
+        "symfony/console": "^6.4 || ^7.1",
+        "symfony/css-selector": "^6.4 || ^7.1",
+        "symfony/dependency-injection": "^6.4 || ^7.1",
+        "symfony/deprecation-contracts": "^2.5 || ^3.0",
+        "symfony/doctrine-bridge": "^6.4 || ^7.1",
+        "symfony/dom-crawler": "^6.4 || ^7.1",
+        "symfony/event-dispatcher": "^6.4 || ^7.1",
+        "symfony/event-dispatcher-contracts": "^2.5 || ^3.0",
+        "symfony/expression-language": "^6.4 || ^7.1",
+        "symfony/filesystem": "^6.4 || ^7.1",
+        "symfony/finder": "^6.4 || ^7.1",
+        "symfony/form": "^6.4 || ^7.1",
+        "symfony/framework-bundle": "^6.4 || ^7.1",
+        "symfony/html-sanitizer": "^6.4 || ^7.1",
+        "symfony/http-client": "^6.4 || ^7.1",
+        "symfony/http-foundation": "^6.4 || ^7.1",
+        "symfony/http-kernel": "^6.4 || ^7.1",
+        "symfony/intl": "^6.4 || ^7.1",
+        "symfony/mailer": "^6.4 || ^7.1",
+        "symfony/mime": "^6.4 || ^7.1",
+        "symfony/options-resolver": "^6.4 || ^7.1",
+        "symfony/process": "^6.4 || ^7.1",
+        "symfony/property-access": "^6.4 || ^7.1",
         "symfony/proxy-manager-bridge": "^5.4 || ^6.0",
-        "symfony/routing": "^5.4 || ^6.0 || ^7.0",
-        "symfony/security-bundle": "^5.4 || ^6.0 || ^7.0",
-        "symfony/security-core": "^5.4 || ^6.0 || ^7.0",
-        "symfony/security-http": "^5.4 || ^6.0 || ^7.0",
-        "symfony/serializer": "^5.4 || ^6.0 || ^7.0",
-        "symfony/string": "^5.4 || ^6.0 || ^7.0",
-        "symfony/translation": "^5.4 || ^6.0 || ^7.0",
-        "symfony/translation-contracts": "^2.3 || ^3.0",
-        "symfony/twig-bundle": "^5.4 || ^6.0 || ^7.0",
-        "symfony/validator": "^5.4 || ^6.0 || ^7.0",
-        "symfony/yaml": "^5.4 || ^6.0 || ^7.0",
-        "toflar/psr6-symfony-http-cache-store": "^3.0 || ^4.0",
+        "symfony/routing": "^6.4 || ^7.1",
+        "symfony/security-bundle": "^6.4 || ^7.1",
+        "symfony/security-core": "^6.4 || ^7.1",
+        "symfony/security-http": "^6.4 || ^7.1",
+        "symfony/serializer": "^6.4 || ^7.1",
+        "symfony/string": "^6.4 || ^7.1",
+        "symfony/translation": "^6.4 || ^7.1",
+        "symfony/translation-contracts": "^2.5 || ^3.0",
+        "symfony/twig-bundle": "^6.4 || ^7.1",
+        "symfony/validator": "^6.4 || ^7.1",
+        "symfony/yaml": "^6.4 || ^7.1",
+        "toflar/psr6-symfony-http-cache-store": "^4.0",
         "twig/twig": "^2.13 || ^3.0",
         "webmozart/assert": "^1.9"
     },
     "require-dev": {
+        "coduo/php-matcher": "^6.0.14",
```

Comment:
accidently added this to require instead of dev dependency move now to dev #7653